apps/epic-react/src/components/video-overlays/blocked-overlay.tsx

```diff
@@ -1,23 +1,367 @@
+            {/*              )}*/}
+            {/*            </Link>*/}
+            {/*          </TooltipTrigger>*/}
+            {/*          <TooltipContent className="text-sm bg-background">*/}
+            {/*            {module.title}*/}
+            {/*          </TooltipContent>*/}
+            {/*        </Tooltip>*/}
+            {/*      )*/}
+            {/*    })}*/}
+            {/*    {showBonuses ? (*/}
+            {/*      <>*/}
+            {/*        <span>+</span>*/}
+            {/*        {bonuses.map((module) => {*/}
+            {/*          return module.state === 'published' ? (*/}
+            {/*            <Tooltip key={module.slug}>*/}
+            {/*              <TooltipTrigger asChild>*/}
+            {/*                <Link*/}
+            {/*                  key={module.slug}*/}
+            {/*                  data-type={module.moduleType}*/}
+            {/*                  href={`/bonuses/${module.slug}`}*/}
+            {/*                  target="_blank"*/}
+            {/*                >*/}
+            {/*                  {(module.image?.url || module.image) && (*/}
+            {/*                    <img*/}
+            {/*                      src={module.image?.url ?? module.image}*/}
+            {/*                      alt={`${module.title} workshop`}*/}
+            {/*                      width={60}*/}
+            {/*                      height={60}*/}
+            {/*                    />*/}
+            {/*                  )}*/}
+            {/*                </Link>*/}
+            {/*              </TooltipTrigger>*/}
+            {/*              <TooltipContent className="text-sm bg-background">*/}
+            {/*                {module.title}*/}
+            {/*              </TooltipContent>*/}
+            {/*            </Tooltip>*/}
+            {/*          ) : (*/}
+            {/*            <Tooltip key={module.slug}>*/}
+            {/*              <TooltipTrigger asChild>*/}
+            {/*                {(module.image?.url || module.image) && (*/}
+            {/*                  <img*/}
+            {/*                    data-type={module.moduleType}*/}
+            {/*                    src={module.image?.url ?? module.image}*/}
+            {/*                    alt={`${module.title} workshop`}*/}
+            {/*                    width={60}*/}
+            {/*                    height={60}*/}
+            {/*                  />*/}
+            {/*                )}*/}
+            {/*              </TooltipTrigger>*/}
+            {/*              <TooltipContent className="text-sm bg-background">*/}
+            {/*                {module.title}*/}
+            {/*              </TooltipContent>*/}
+            {/*            </Tooltip>*/}
+            {/*          )*/}
+            {/*        })}*/}
+            {/*      </>*/}
+            {/*    ) : null}*/}
+            {/*  </TooltipProvider>*/}
+            {/*</div>*/}
```

Comment:
this has a lot more content than Total TypeScript so it looked bad plus all the images were broken